src/SignalR/common/Http.Connections/src/Internal/Transports/LongPollingServerTransport.cs

```diff
@@ -18,6 +18,10 @@ internal class LongPollingServerTransport : IHttpTransport
     private readonly CancellationToken _timeoutToken;
     private readonly HttpConnectionContext? _connection;
 
+    private const string HeaderValueNoCache = "no-cache";
+    private const string HeaderValueNoCacheNoStoreMustRevalidate = "no-cache, no-store, must-revalidate";
+    private const string HeaderValueEpochDate = "Thu, 01 Jan 1970 00:00:00 GMT";
```

Comment:
@Tratcher do you think it's worth sharing the list of response headers and values needed to disable caching with `CookieAuthenticationHandler`? Or do you think this is simple enough and unlikely enough to change that we can just keep it? I think we should probably at least add a comment to both saying that `LongPollingServerTransport.cs` and `CookieAuthenticationHandler.cs` should keep the cache-disabling response headers in sync.

Reply:
Shrug, the code in cookie auth hasn't changed in 5+ years, and the values aren't complicated. This is more of a best practice.

Reply:
Not worth it, comment is good enough